examples/compare_product_prices/README.md

```diff
@@ -0,0 +1,16 @@
+# Example script: comparing product price across websites with AgentQL
+
+This example demonstrates how to compare product prices across websites with query_data() method.
+
+## Run the script
+
+* [Install AgentQL SDK](https://docs.agentql.com/docs/installation/sdk-installation)
+* Save this python file locally as **compare_product_prices.py**
+* Run the following command from the project's folder:
+```bash
+python3 compare_product_prices.py
+```
+
+## Play with it
```

Comment:
This heading can be confusing. First heading is "Run the script", second "Play with it" reads like "Play with the script". What about "Play with the query"?

Reply:
That is a great suggestion!